pkg/secrets/check_rights_windows_test.go

```diff
@@ -48,71 +48,84 @@ func TestSpaceInPath(t *testing.T) {
 	require.Nil(t, checkRights(tmpFile.Name(), false))
 }
 
-func TestCheckRights(t *testing.T) {
-	// default options
-	allowGroupExec := false
-
+func TestCheckRightsDoesNotExists(t *testing.T) {
 	// file does not exist
-	require.NotNil(t, checkRights("/does not exists", allowGroupExec))
+	require.NotNil(t, checkRights("/does not exists", false))
+}
 
-	// missing current user
+func TestCheckRightsMissingCurrentUser(t *testing.T) {
 	tmpfile, err := os.CreateTemp("", "agent-collector-test")
 	require.Nil(t, err)
 	defer os.Remove(tmpfile.Name())
 
-	exec.Command("powershell", "test/setAcl.ps1",
+	err = exec.Command("powershell", "test/setAcl.ps1",
 		"-file", tmpfile.Name(),
 		"-removeAllUser", "1",
 		"-removeAdmin", "0",
 		"-removeLocalSystem", "0",
 		"-addDDuser", "0").Run()
-	assert.NotNil(t, checkRights(tmpfile.Name(), allowGroupExec))
+	require.NoError(t, err)
+	assert.NotNil(t, checkRights(tmpfile.Name(), false))
+}
 
-	// missing localSystem
+func TestCheckRightsMissingLocalSystem(t *testing.T) {
 	tmpfile, err = os.CreateTemp("", "agent-collector-test")
```

Comment:
```suggestion
	tmpfile, err := os.CreateTemp("", "agent-collector-test")
```
Lots of similar failures below too